stable/distribution/README.md

```diff
@@ -357,8 +396,10 @@ The following table lists the configurable parameters of the distribution chart
 | `distributor.loggersResources.limits.cpu`       | Distributor loggers cpu limit                                          |                                                                    |
 | `router.name`                                   | Router name                                                            | `router`                                                           |
 | `router.image.pullPolicy`                       | Container pull policy                                                  | `IfNotPresent`                                                     |
-| `router.image.repository`                       | Container image                                                        | `docker.bintray.io/jfrog/router`                                     |
+| `router.image.repository`                       | Container image                                                        | `docker.bintray.io/jfrog/router`                                   |
 | `router.image.version`                          | Container image tag                                                    | `.Chart.AppVersion`                                                |
+| `router.tlsEnabled`                             | Enable TLS connection                                                  | `false`                                                            |
+
       
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
```

Comment:
Please add the `distribution.configMap` parameter to the README

Reply:
Done